### PR TITLE
Update gradle command in README.md

### PR DESCRIPTION
**Pull request by maya**
Seems like this is the correct command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,14 +79,14 @@ Running spring app locally and db in docker
 
 1. `cd backend`
 1. Run `docker-compose up -d db`
-1. Run `gradle bootRun --args='--spring.profiles.active=dev'`
+1. Run `./gradle bootRun --args='--spring.profiles.active=dev'`
```
Comment (nickclyde) [inline README.md:82]: These should actually be `./gradlew`. Technically `gradle` will work if you have gradle installed on your system, but `./gradlew` is a wrapper script (that's what the w stands for) that downloads a specific gradle version to your system if it doesn't already have it, so it is the more compatible command

Comment (maya) [reply README.md]: Update in 8bf68e6

**Review by nickclyde — APPROVED** [review]
:+1:

**Review by benwarfield-usds — APPROVED** [review]
Yes. Yes it is.

That does partially explain why people keep having that problem recently...

**Pull request merged**